### ISSUE TITLE
Typo in documentation

### ISSUE THREAD

**Issue by sgruetter** [brianc/node-postgres#1489]
Hi, there seems to be a typo [in the transactions documentation](https://node-postgres.com/features/transactions), in the first paragraph: 
```
Because node-postgres strives to be low level an un-opinionated it doesn't provide
                                               ^ should be 'and' 
```
And by the way thanks for the great library 😃

**Comment by DarkMukke** [comment]
Another typo on https://node-postgres.com/features/pooling

```
// promise - checkout a client
pool.connect()
  .then(client => {
    return client.query('SELECT * FROM users WHERE id = $1', [1])
      .then(res => {
        client.release()
        console.log(res.rows[0])
      })
      .catch(e => {
        client.release()
        console.log(err.stack)
      })
  })
```

should be 

```
// promise - checkout a client
pool.connect()
  .then(client => {
    return client.query('SELECT * FROM users WHERE id = $1', [1])
      .then(res => {
        client.release()
        console.log(res.rows[0])
      })
      .catch(err => {
        client.release()
        console.log(err.stack)
      })
  })
```

note the ` .catch(err => {` bit

**Comment by charmander** [comment]
“and” was fixed at some point, the other is #1857.
